.github/workflows/emscripten.yml

```diff
@@ -5,6 +5,26 @@ on:
     branches:
       - main
       - maintenance/**
+  # Note: this workflow gets triggered on the same schedule as the
+  # wheels.yml workflow, with the exception that this workflow runs
+  # the test suite for the Pyodide wheel too, prior to uploading it.
+  #
+  # Run on schedule to upload to Anaconda.org
+  schedule:
+    #        ┌───────────── minute (0 - 59)
+    #        │  ┌───────────── hour (0 - 23)
+    #        │  │ ┌───────────── day of the month (1 - 31)
+    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │  │ │ │ │
+    - cron: "42 2 * * SUN,WED"
+  workflow_dispatch:
+    inputs:
+      push_wheels:
+        description: >
+          Push wheels to Anaconda.org if the build is successful, can be 'true' or 'false'. Default is 'false'. Warning: this will overwrite existing wheels.
```

Comment:
Could you shorten this? Maybe something like
```
Push wheels to Anaconda.org if the build succeeds
```
There rest could be a comment in an appropriate spot.

Reply:
Sure! Shortened this in [31411af](https://github.com/numpy/numpy/pull/26134/commits/31411afd7c88ea3bdedf8f8afaf321102ba742a8), I can rebase/squash as necessary when the PR is a step away from merging.